tests/config_schema_examples/example1.json

```diff
@@ -0,0 +1,23 @@
+{
+    "editor": "vi",
+    "queues": [
+        {
```

Comment:
These should be updated with the executor format, and the files should be named to give more meaningful insight about their content. "Example1" tells me nothing, but `tests/config_examples/local-example.json` or `tests/config_examples/slurm-example.json` is meaningful.

Reply:
I will fix this in next commit